darts/timeseries.py

```diff
@@ -758,20 +774,21 @@ def from_dataframe(df: pd.DataFrame,
         TimeSeries
             A univariate or multivariate TimeSeries constructed from the inputs.
```

Comment:
Here too, you should document dummy_index

Reply:
Done